types.go

```diff
@@ -1,16 +1,14 @@
 package frontman
 
-import (
-	"encoding/json"
-)
-
 type ServiceName string
 
 const (
-	ProtocolICMP = "icmp"
-	ProtocolTCP  = "tcp"
-	ProtocolUDP  = "udp"
-	ProtocolSSL  = "ssl"
+	ProtocolICMP   = "icmp"
+	ProtocolTCP    = "tcp"
+	ProtocolUDP    = "udp"
+	ProtocolSSL    = "ssl"
+	ProtocolDNSUDP = "dns.udp"
```

Comment:
I'm kind of confused because of these new Protocol* consts.

We do already have handlers for `protocol = "udp"`  and `protocol = "tcp"` which handle different services (`http`, `ssh`, `sip`, `iax2`, etc.). Maybe reusing this structure can simplify the code and make the input.json format more clear to the end-user?
Using the already documented checks I would expect new checks to be like:
1) ` "check": { "connect": "1.1.1.1", "protocol": "udp", "service": "dns"}`
2) ` "check": { "connect": "1.1.1.1", "protocol": "tcp", "service": "dns"}`

Not sure, maybe you have already discussed this with Thorsten, though. There is no input.json example in task description.

Anyway, it's worth adding new checks to check list in README.md

Reply:
I like your suggestion. Reworking PR now

Reply:
Reworked